Makefile

```diff
@@ -15,10 +15,3 @@ proto:
 .PHONY: generate
 generate:
 	go generate ./...
-
-.PHONY: image-push-all
-image-push-all:
```

Comment:
don't delete the make target, put `./hack/update_examples -bp -t test` command in the target?

Reply:
it is a handy command to have in make

Reply:
+1. @ayildirim21 please also put a comment on the target to mention where we are pushing to(quay.io).